lintrunner_adapters/adapters/add_trailing_comma_linter.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import argparse
+import concurrent.futures
+import logging
+import os
+import sys
+from typing import List
+
+from add_trailing_comma._main import _fix_src
+
+from lintrunner_adapters import LintMessage, LintSeverity, add_default_options
+
+LINTER_CODE = "ADD-TRAILING-COMMA"
+
+
+def format_error_message(filename: str, err: Exception) -> LintMessage:
+    return LintMessage(
+        path=filename,
+        line=None,
+        char=None,
+        code=LINTER_CODE,
+        severity=LintSeverity.ADVICE,
+        name="command-failed",
+        original=None,
+        replacement=None,
+        description=(f"Failed due to {err.__class__.__name__}:\n{err}"),
+    )
+
+
+def check_file(
+    filename: str,
+    min_version: tuple[int, ...],
+) -> List[LintMessage]:
+    with open(filename, "rb") as fb:
+        contents_bytes = fb.read()
+
+    try:
+        original = contents_text = contents_bytes.decode("utf-8")
+        replacement = _fix_src(contents_text, min_version)
+
+        if original == replacement:
+            return []
+
+        return [
+            LintMessage(
+                path=filename,
+                line=None,
+                char=None,
+                code=LINTER_CODE,
+                severity=LintSeverity.WARNING,
+                name="format",
+                original=original,
+                replacement=replacement,
+                description="Run `lintrunner -a` to apply this patch.",
+            )
+        ]
+    except Exception as err:
+        return [format_error_message(filename, err)]
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description=f"add-trailing-comma wrapper linter. Linter code: {LINTER_CODE}",
+        fromfile_prefix_chars="@",
+    )
+    parser.add_argument(
+        "--py35-plus",
```

Comment:
Python 3.6 etc. are obsolete versions. Should the default be a higher version e.g. 3.7?

Reply:
Updated